acceptance/pre_suites/90_setup_ras_demo.rb

```diff
@@ -0,0 +1,26 @@
+test_name "install required gems on controller" do
```

Comment:
test name could be better.  Like what are they required for, and why aren't they needed for anything besides the demo.  Might not be able to fit that all in the name, but a comment could share the logic.
I think the reason is because it enables you to build the gem on the controller.